src/analysis/scripts/export_connectivity.sh

```diff
@@ -75,6 +75,35 @@ function ec_export_table_csv() {
          -c "\COPY ${EXPORT_TABLENAME} TO '${FILENAME}' WITH (FORMAT CSV, HEADER)"
 }
 
+function ec_export_table_geojson() {
+  OUTPUT_DIR="${1}"
+  EXPORT_TABLENAME="${2}"
+
+  FILENAME="${OUTPUT_DIR}/${EXPORT_TABLENAME}.geojson"
+  ogr2ogr -f GeoJSON "${FILENAME}" \
+          -t_srs EPSG:4326 \
```

Comment:
I'm inclined to say we should err on the side of explicitness--we're confident we know what SRID the tables are in, so might as well tell `ogr2ogr` and not rely on it to figure it out.  Then again if the errors you were seeing were actually related to something else there might be advantages to letting it autodetect, e.g. in case we do ever end up making tables in other coordinate systems for some reason.